dogs/src/app/home-page/home-page.component.html

```diff
@@ -1,6 +1,6 @@
 <div>
 	<ul>
 		<!-- Deocamdata doar afisez lista, voi face ruta in urmatorul pas -->
```

Comment:
try <!--TODO this is an example-->
and install Highlight extension in order to mark it